### ISSUE TITLE
Issue starting the integration 'image_width referenced before assignment'

### ISSUE THREAD

**Issue by tnk-rin** [briis/unifiprotect#264]
I followed the installation instructions but i keep getting this error when the integration starts

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 293, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/config/custom_components/unifiprotect/__init__.py", line 99, in async_setup_entry
    await protect_data.async_setup()
  File "/config/custom_components/unifiprotect/data.py", line 32, in async_setup
    await self.async_refresh()
  File "/config/custom_components/unifiprotect/data.py", line 48, in async_refresh
    await self._protectserver.update(
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 123, in update
    await self._get_device_list(not self.ws_connection)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 314, in _get_device_list
    self._process_cameras_json(json_response, server_id, include_events)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 335, in _process_cameras_json
    process_camera(
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_data.py", line 349, in process_camera
    "image_width": image_width,
UnboundLocalError: local variable 'image_width' referenced before assignment
```

**Comment by briis** [comment]
What version of the Integration are you running?

**Comment by tnk-rin** [comment]
v0.8.9

**Comment by briis** [comment]
This is now fixed, and will be in the 0.9.0 release. Thanks for reporting.
